docs/topics/forms/modelforms.txt

```diff
@@ -1230,10 +1230,53 @@ of a model. Here's how you can do that::
 
 Notice how we pass ``instance`` in both the ``POST`` and ``GET`` cases.
 
+Inline formsets and ``related_name``
+------------------------------------
+
+If the foreign key on your child model has a ``related_name`` to the parent,
+this is set to the formset's prefix when using ``inlineformset_factory``.
+
+For example, consider the following models and formset::
+
+    from django.db import models
+    from django.forms import inlineformset_factory
+
+    class Author(models.Model):
+        name = models.CharField(max_length=100)
+
+    class Book(models.Model):
+        author = models.ForeignKey(Author, on_delete=models.CASCADE, related_name='books')
+        title = models.CharField(max_length=100)
+
+    BookFormSet = inlineformset_factory(Author, Book, fields=('title',), extra=1)
+
+``BookFormSet`` above will have its ``prefix`` attribute set to 'books'. This will be added to each
+field's 'name' and 'id' attributes in the template::
+
+    >>> formset = BookFormSet()
+    >>> print(formset)
+    <input type="hidden" name="books-TOTAL_FORMS" value="3" id="id_books-TOTAL_FORMS" /><input type="hidden" name="books-INITIAL_FORMS" value="0" id="id_books-INITIAL_FORMS" /><input type="hidden" name="books-MIN_NUM_FORMS" value="0" id="id_books-MIN_NUM_FORMS" /><input type="hidden" name="books-MAX_NUM_FORMS" value="1000" id="id_books-MAX_NUM_FORMS" />
```

Comment:
Is there a less messy way to demonstrate that the formset has a prefix? Obviously you could just print the `prefix` member, but that might not get the point across. I looked for other places that the `prefix` element is used and I couldn't see anything suitable.

Actually, the example below of prefixing the data before passing it to the form is probably as good a demonstration as any. I'd be tempted just to print the prefix at this point.